Database/funds.json

```diff
@@ -0,0 +1,6 @@
+[
+  { "id": "1", "name": "Cushon Equities Fund" },
```

Comment:
An id is usually an integer - and in a real-world scenario, it will come from a database as an autoincrement.

Saying this - it's a security risk to ever expose an ID to a client/via api. Change all your ids to a UUID. This is a good article here for this: https://1kevinson.com/why-using-uuid-instead-of-ids-in-your-java-project/

Consider creating your own UUID generator OR using a package. There are trade-offs for build vs using an npm package